.gitignore

```diff
@@ -11,3 +11,4 @@ phpunit.xml
 /.phpcs-cache
 /box.phar
 /box.json
+/.idea
```

Comment:
This should be present in your global gitignore. Different people might use different tools and we definitely don't want to add all possible configuration files here.

Reply:
see https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore for the doc about that.

@lcobucci I suggest you to include this link in your answers about that point in the future. It makes it easier for people not experienced with git to solve it :smile:

Reply:
I added a comment instead, hope you like it